spec/models/vm_or_template_spec.rb

```diff
@@ -335,6 +335,19 @@
     end
   end
 
+  context "#resource_group" do
+    before do
```

Comment:
Again, sanity testing in case someone adds scopes/filters/whatever that conflict with what I'm expecting.